aptos-move/move-examples/fungible_asset/sources/managed_fungible_asset.move

```diff
@@ -26,7 +26,7 @@ module fungible_asset::managed_fungible_asset {
     /// Initialize metadata object and store the refs.
     fun init_module(admin: &signer) {
         let constructor_ref = &object::create_named_object(admin, ASSET_SYMBOL);
-        primary_wallet::create_primary_wallet_enabled_fungible_asset(
+        primary_store::create_primary_wallet_enabled_fungible_asset(
```

Comment:
This looks broken. There's no create_primary_wallet_enabled_fungible_asset anymore

Reply:
my bad.